src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class RightTriangle extends Figure {
+    private static final String type = "triangle";
+    private final int firstLeg;
+    private final int secondLeg;
+
+    public RightTriangle(int firstLeg, int secondLeg, String color) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        this.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) firstLeg * secondLeg * 0.5;
```

Comment:
Magic number '0.5' should be extracted as a constant field to improve code readability and maintainability.